README.md

```diff
@@ -9,11 +9,10 @@ My first project during the first week of training in Microverse.
 - CSS
 - Linters
 ​
-​
-​
 ## Getting Started
 ​
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/Epaltechs/Hello-World.git
+cd Hello-World
```

Comment:
- [ ] OPTIONAL: You have done a good job adding how to clone the repository, but currently it lacks information and requires proper layout. Try this: 
```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/Epaltechs/Hello-World.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/ShoiraTa/blog_app.git`.
- go to the repository folder in your command prompt `cd Hello-World`

```

Reply:
I appreciate it so much.